climakitae/data_loaders.py

```diff
@@ -118,26 +133,35 @@ def _read_from_catalog(selections, location):
         ds_region = None
 
     if selections.append_historical:
-        one_scenario = 'Historical Climate'
+        one_scenario = "Historical Climate"
         files_by_scenario = _get_file_list(selections, one_scenario)
         historical = _open_and_concat(files_by_scenario, selections, ds_region)
-        
-    all_files = xr.Dataset()
+
+    all_files_list = []
     for one_scenario in selections.scenario:
         if selections.append_historical:
             if "SSP" in one_scenario:
```

Comment:
when I only have 'Append Historical' checked, I get
`TypeError: argument of type 'NoneType' is not iterable`

This makes sense (can't append to nothing!), but maybe amend Append Historical to "Append Historical (select SSP and/or reconstruction)". Or is there a way to only have it appear if another thing is checked in the first place? There could also be an error that explicitly states that another box needs to be checked.

Reply:
Adding another assert in data_loaders._read_from_catalog

Reply:
So at least it gives an error that will make sense to the user now: "Please select at least one..."